src/styles/blocks/header.scss

```diff
@@ -0,0 +1,43 @@
+.header {
+  //for bg image to fill the whole screen
+  //background-position: center;
```

Comment:
It looks like you commented out the background-position and background-size properties. If you don't need them, consider removing these lines to keep the code cleaner. 🧹